ci/build_linux.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
```

Comment:
I have just copied the scripts here for simplicity, but maybe we should lift the shared parts out to a build_base.sh, happy to get your thoughts

Reply:
I think moving the shared parts out makes sense